scripts/api-compatibility-check/.baseapi

```diff
@@ -0,0 +1 @@
+v10.1.0
```

Comment:
Why not v10.0.0?

Reply:
There is a set of `@_spi` related changes that break API.